mono/unit-tests/test-path.c

```diff
@@ -0,0 +1,79 @@
+/*
+ * test-path.c
+ */
+
+#include "config.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include "glib.h"
+
+gboolean mono_host_win32;
+#undef HOST_WIN32
+#define HOST_WIN32 mono_host_win32
```

Comment:
As mentioned before https://github.com/mono/mono/pull/9930#discussion_r245003497 we're not going to take this hack.

Reply:
Testability technique?